pyblish_qml/control.py

```diff
@@ -17,6 +17,12 @@
 from pyblish_qml import settings
 
 
+def toggle_instance(instance, new_value, old_value):
+    instance.data['publish'] = bool(new_value)
+
+pyblish.api.register_callback("instanceToggled", toggle_instance)
```

Comment:
I'm not a fan of having modules produce side-effects on import. How about putting this in the [main](https://github.com/tokejepsen/pyblish-qml/blob/1be49764f34605d532b68d600f097e6275475a08/pyblish_qml/app.py#L226) instead, so we can safely import it without worry?